bands/site_sections/band_admin.py

```diff
@@ -24,6 +24,21 @@ def mark_as_band(self, session, group_id):
             'message': '{} has been marked as a band'.format(group.name)
         }
 
+    @ajax
+    def remove_as_band(self, session, group_id):
+        group = session.group(group_id)
+        if not group.leader:
+            return {'message': '{} does not have an assigned group leader'.format(group.name)}
```

Comment:
Right, but the problem is the following situation:
- Admin marks a group as a band.
- Group's leader is removed.
- Admin tries to un-mark the group as a band, but cannot.
- Admin now needs to make someone the group leader before they can be un-marked as a band.

There's probably no good reason to require that last step, so we should probably remove this.